pkg/api/unversioned.go

```diff
@@ -49,3 +53,12 @@ func FieldSelectorQueryParam(version string) string {
 	}
 	return "field-selector"
 }
+
+// String returns available api versions as a human-friendly version string.
+func (apiVersions APIVersions) String() string {
+	return strings.Join(apiVersions.Versions, ",")
+}
+
+func (apiVersions APIVersions) GoString() string {
```

Comment:
@jlowdermilk 
GoString is called [here](https://github.com/you-n-g/kubernetes/blob/master/pkg/kubectl/version.go#L51)
this is a little more friendly

Otherwise it will print the APIVersions struct literally maybe like `unversioned.APIVersions{Versions:[v1beta1 v1beta2 v1beta3]}`
`